workers-site/index.js

```diff
@@ -11,23 +8,35 @@ addEventListener('fetch', event => {
     event.respondWith(handleEvent(event))
   } catch (e) {
     if (DEBUG) {
-    return event.respondWith(new Response(e.message || e.toString(), {
-      status: 404,
-    }))
-  }
+      return event.respondWith(
+        new Response(e.message || e.toString(), {
+          status: 500,
+        }),
+      )
+    }
     event.respondWith(new Response('Internal Error', { status: 500 }))
   }
 })
 
 async function handleEvent(event) {
   const url = new URL(event.request.url)
+    let options = {
+      // customize how incoming requests will map to assets
+      mapRequestToAsset: request => {
+        // compute the key used by default (e.g. / -> index.html)
+        let defaultAssetKey = mapRequestToAsset(request)
+        let url = new URL(defaultAssetKey.url)
+        // strip the prefix from looking up /docs for all files
+        url.pathname = url.pathname.replace(/^\/docs/, '/')
```

Comment:
This seems super specific to a docs use-case, is this intended to show what you _can_ do? Wouldn't this code be placed in every generated project that uses this template?